addons/core/tests/integration/components/image-test.js

```diff
@@ -13,7 +13,7 @@ module('Integration | Component | Image', function (hooks) {
 
   test('it renders', async function (assert) {
     assert.expect(1);
-    await render(hbs`<Image @name="logo-app" />`);
+    await render(hbs`<Image @name='logo-app' />`);
```

Comment:
I have noticed we are changing from double quotes (`"`) to single quotes (`'`) within the test files. What is the reason behind this change? i.e on Admin UI we use double quotes.

Reply:
@calcaide looking into this.